cluster/juju/layers/kubernetes-master/actions/create-rbd-pv

```diff
@@ -203,6 +223,17 @@ def validate_relation():
     return True
 
 
+def validate_space(size):
+    if action_get_or_default('skip-size-check'):
+        return True
+    available_space = get_available_space()
+    if available_space < size:
+        msg = 'Unable to allocate RBD of size {}MB, only {}MB are available.'
+        action_fail(msg.format(size, available_space))
```

Comment:
Seems weird to fail the action this deep in the call stack... but I see you're using the space values in the messaging here. I'm +1 to this